src/main/java/org/mastodon/ui/keymap/KeymapManager.java

```diff
@@ -229,9 +230,9 @@ private static Yaml createYaml()
 	{
 		final DumperOptions dumperOptions = new DumperOptions();
 		dumperOptions.setDefaultFlowStyle( DumperOptions.FlowStyle.BLOCK );
-		final Representer representer = new Representer();
+		final Representer representer = new Representer( new DumperOptions() );
```

Comment:
There is a `dumperOptions` constructed 2 lines above. Why not use that instead of a new one?